distributions/test/src/test/java/com/connexta/replication/distributions/test/ClassicReplicationE2E.java

```diff
@@ -53,42 +55,56 @@
 import org.testcontainers.containers.Network;
 import org.testcontainers.containers.output.Slf4jLogConsumer;
 
-public class ReplicationE2E {
+public class ClassicReplicationE2E {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(ReplicationE2E.class);
+  private static final Logger LOGGER = LoggerFactory.getLogger(ClassicReplicationE2E.class);
 
   // used to collect the replication service logs and print them with the test logs
   private static final Slf4jLogConsumer logConsumer = new Slf4jLogConsumer(LOGGER);
 
   @ClassRule public static Network network = Network.newNetwork();
 
-  @ClassRule public static GenericContainer solr = defaultSolrContainer(network);
+  private static final String sourceUrl = System.getenv("REPLICATION_SRC");
+  private static final String destinationUrl = System.getenv("REPLICATION_DEST");
+
+  public static GenericContainer solr = defaultSolrContainer(network);
 
-  @ClassRule public static GenericContainer replication = defaultReplicationContainer(network);
+  public static GenericContainer replication = defaultReplicationContainer(network, "Classic");
 
   private static String solrUrl;
-  private static final String sourceUrl = System.getenv("REPLICATION_SRC");
-  private static final String destinationUrl = System.getenv("REPLICATION_DEST");
 
   private final Set<String> idsToDelete = new HashSet<>();
 
   @BeforeClass
   public static void beforeClass() {
+    solr.start();
     assertTrue("Source and destination URLs not set!", sourceUrl != null && destinationUrl != null);
     LOGGER.info("Source URL: {}", sourceUrl);
     LOGGER.info("Destination URL: {}", destinationUrl);
     RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();
-    replication.followOutput(logConsumer);
     solrUrl = "http://" + solr.getContainerIpAddress() + ":" + solr.getFirstMappedPort();
     LOGGER.info("Solr URL: {}", solrUrl);
+    String sites =
+        getFileAsText("sites.json", Map.of("source", sourceUrl, "destination", destinationUrl));
+    postToSolrCore(solrUrl, SITE, sites);
```

Comment:
Replication will now break if it can't find the local site in solr so we have to make sure it's created before we start the replication container.